### DISCUSSION TITLE
Read array values

### DISCUSSION THREAD

**Discussion by rupertbauernfeind** [danielhrisca/asammdf#673] [Q&A]
Hello everybody,

I have created a mf4 logfile with Vector CANoe and it converted the Arrays as 35GB empty channel group.
I am not able to read the array values. Any other non-array values can be read out fine.

Do you have any tips, how I can convert the empty channel information?
![image](https://user-images.githubusercontent.com/26060727/159265472-6bf84229-d45c-4794-a579-aceaa1cba087.png)

**Comment by danielhrisca** [comment]
It's not clear what you do there. Please give more details
